src/design-system/color/AccentColorContext.tsx

```diff
@@ -1,9 +1,9 @@
 import chroma from 'chroma-js';
 import React, { createContext, ReactNode, useMemo } from 'react';
-import { BackgroundColorValue, defaultAccentColor } from './palettes';
+import { BackgroundColorValue } from './palettes';
 
-export const AccentColorContext = createContext<BackgroundColorValue>(
-  defaultAccentColor
+export const AccentColorContext = createContext<BackgroundColorValue | null>(
+  null
```

Comment:
The default accent color can no longer be provided by default from the provider since the value now depends on the color mode.